tornettools/generate_tgen.py

```diff
@@ -249,26 +249,69 @@ def __convert_privcount_key_to_tgen_key(str):
     else:
         return str
 
-def get_servers(args, n_clients):
+def __calculate_number_of_servers(args, n_public_clients, n_hs_clients):
+    # the number of servers will be a fraction of the number of clients
+    n_public_servers = n_public_clients * args.server_scale
+    n_hs_servers = n_hs_clients * args.server_scale
+    n_servers = n_public_servers + n_hs_servers
+
+    # if we don't have enough servers, scale both server counts
+    if n_servers < TGEN_SERVER_MIN_COUNT:
+        n_public_servers *= TGEN_SERVER_MIN_COUNT / n_servers
+        n_hs_servers *= TGEN_SERVER_MIN_COUNT / n_servers
+        n_servers = n_public_servers + n_hs_servers
```

Comment:
This line is unnecessary, since we write `n_servers` again below before we read it.

Reply:
Removed this line.